concourse/pipeline.yml

```diff
@@ -152,6 +176,12 @@ jobs:
           SECRET_KEY_BASE: ((secret-key-base-staging))
           WEBSITE_ROOT: https://www.staging.publishing.service.gov.uk
           WORKER_INSTANCES: 1
+        on_success:
```

Comment:
Do we also want the `path:` thing in here (and the deploy to production below) so that the annotation shows up as a region, rather than two points?

Reply:
🤔 Interesting, what would that look like? Do you have a link to anything similar?
So this would be instead of two "goalposts" of annotation lines we get a shaded area in between to see when a deploy is underway?

🤔 I'd need to move that out of the `on_success` section then, right? Otherwise we'd get regions that ar begun and never ended on failure?

Reply:
I don't know what it'd look like, but I imagine a shaded box drawn over a region on the graph.

The region is created by setting the finish point, so if a deploy starts and then fails we'd see a mark for the start of deploy and no region.

Reply:
Ace, done thanks for the improvement!